posts/2024-05-10_new_admiral_ex.../new_admiral_extension_packages_admiralpeds_coming_soon.qmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "New {admiral} extension packages: {admiralpeds} & more coming soon!"
+author:
+  - name: Fanny Gautier
+  - name: Lina Patil
+description: "We are very excited to announce that new extension packages are expected very soon: {admiralpeds} which is a pediatrics extension, and another extension for obesity in clinical trials"
+date: "2024-05-10"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Metadata, ADaM, Community]
+# feel free to change the image
+image: "coming_soon.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-05-10_new_admiral_ex..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+[![](admiralpeds.png){fig-align="center" width="220"}](https://pharmaverse.github.io/admiralpeds/index.html)
+
+📢 We are not just adopting new tools but fostering a culture of collaboration, innovation and a commitment to continuous learning. We are thus thrilled to update you on the latest developments from our admiral family team: [`admiralpeds 0.1`](https://pharmaverse.github.io/admiralpeds/index.html) is __coming soon__ [on CRAN](https://cran.r-project.org) !
+
+`{admiralpeds}` is a pediatrics extension package for ADaM in R Asset Library `{admiral}`. Its first release is planned by July 2024.
+
+# Collaboration
+
+7 companies (Roche, Pfizer, Amgen, BMS, J&J, Novartis, and Cytel) and David Freedman, a passionate consultant retired from CDC, are working together to create this extension 🤝.
+
+# Purpose
+
+The purpose of this new extension is to provide a complementary (to `{admiral}`) toolbox that enables users to develop specifics ADaMs datasets for pediatric clinical trials.
+Initially the package will focus on child growth/development charts: [WHO and CDC growth charts](https://www.cdc.gov/growthcharts/who_charts.htm), among infants and children ages birth to 19 years included. 
+
+# New Features
```

Comment:
@Fanny-Gautier @Lina2689 Can you also mention anything else the package provides? Templates, test data, website vignettes etc. Thanks.